fastlane/lib/fastlane/actions/cocoapods.rb

```diff
@@ -72,66 +72,60 @@ def self.available_options
-                                       is_string: false,
+                                       type: Boolean,
                                        default_value: false),
           FastlaneCore::ConfigItem.new(key: :silent,
                                        env_name: "FL_COCOAPODS_SILENT",
                                        description: "Execute command without logging output",
-                                       is_string: false,
+                                       type: Boolean,
                                        default_value: false),
           FastlaneCore::ConfigItem.new(key: :verbose,
                                        env_name: "FL_COCOAPODS_VERBOSE",
                                        description: "Show more debugging information",
-                                       is_string: false,
+                                       type: Boolean,
                                        default_value: false),
           FastlaneCore::ConfigItem.new(key: :ansi,
                                        env_name: "FL_COCOAPODS_ANSI",
                                        description: "Show output with ANSI codes",
-                                       is_string: false,
+                                       type: Boolean,
                                        default_value: true),
           FastlaneCore::ConfigItem.new(key: :use_bundle_exec,
                                        env_name: "FL_COCOAPODS_USE_BUNDLE_EXEC",
                                        description: "Use bundle exec when there is a Gemfile presented",
-                                       is_string: false,
+                                       type: Boolean,
                                        default_value: true),
           FastlaneCore::ConfigItem.new(key: :podfile,
                                        env_name: "FL_COCOAPODS_PODFILE",
                                        description: "Explicitly specify the path to the Cocoapods' Podfile. You can either set it to the Podfile's path or to the folder containing the Podfile file",
                                        optional: true,
-                                       is_string: true,
                                        verify_block: proc do |value|
                                          UI.user_error!("Could not find Podfile") unless File.exist?(value) || Helper.test?
                                        end),
           FastlaneCore::ConfigItem.new(key: :error_callback,
                                        description: 'A callback invoked with the command output if there is a non-zero exit status',
                                        optional: true,
-                                       is_string: false,
-                                       type: :string_callback,
-                                       default_value: nil),
```

Comment:
This `string_callback` 🤯 
Q: do I need to keep `default_value: nil` for this type? 🤔

Reply:
`optional: true` should be all that we need here 💪